### DISCUSSION TITLE
Activity log summary and stats or visualizations

### DISCUSSION THREAD

**Discussion by amanpatel** [spatie/laravel-activitylog#802] [Q&A]
Is there any package that can take the data in `activity_log` and extract summaries from it? Perhaps even allowing for filtering along the way and visualizations of the activity.

Could be super useful!

**Comment by Gummibeer** [comment]
No package I know.

What do you expect? Which data should be aggregated?

**Reply by amanpatel** [reply]
Great question. Here's one example. Let's say if you are recording "views" for resources like documents or video downloads in a CMS type website. A log summarization tool could perhaps be able to show number of entries by various attributes of the user account.

But as I write this, It does occur to me that doing this might not be a very general solution. I'm sure activity log would differ by a lot.
